common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -241,10 +270,13 @@ function (Iterator) {
          * @param {jquery Event} event
          */
         clickLinkHandler: function (event) {
-            var speed = $(event.currentTarget).parent().data('speed');
-
-            this.closeMenu();
+            var el = $(event.currentTarget).parent(),
+                speed = $(el).data('speed');
+                
+            this.resetActiveSpeed();
+            this.setActiveSpeed(speed);
```

Comment:
The work here is mainly front-end. In short, we needed to better indicate the selected state for high contrast screens. These new functions add necessary style classes to achieve this. More information is in the description at the top of this PR. Let me know if you have other questions.